setup.py

```diff
@@ -133,7 +133,7 @@ def run(self):
     description="Git for data scientists - manage your code and data together",
     long_description=open("README.rst", "r").read(),
     author="Dmitry Petrov",
-    author_email="dmitry@dataversioncontrol.com",
+    author_email="dmitry@iterative.ai",
```

Comment:
@dmpetrov please note this change. Is it correct?

Reply:
yep, it's good. resolving this.

Reply:
may be it's even better to put dmitry@dvc.org

Reply:
OK. Updated in a bunch of places in 039f196d.